istioctl/cmd/revision.go

```diff
@@ -116,15 +116,15 @@ func revisionCommand() *cobra.Command {
 func revisionDescribeCommand() *cobra.Command {
 	describeCmd := &cobra.Command{
 		Use: "describe",
-		Example: `  # View the details of a revision named 'canary'    
+		Example: `  # View the details of a revision named 'canary'
```

Comment:
Is there any problem with these whitespaces? I'm sure these are detected by IDE and there might be a lot of whitespace like this in the codebase.